packages/edit-site/src/components/navigation-sidebar/navigation-panel/constants.js

```diff
@@ -30,3 +35,35 @@ export const MENU_TEMPLATES_PAGES = 'templates-pages';
 export const MENU_TEMPLATES_POSTS = 'templates-posts';
 
 export const SEARCH_DEBOUNCE_IN_MS = 75;
+
+export const MENU_TEMPLATE_PARTS_HEADERS = 'template-parts-headers';
+export const MENU_TEMPLATE_PARTS_FOOTERS = 'template-parts-footers';
+export const MENU_TEMPLATE_PARTS_SIDEBARS = 'template-parts-sidebars';
+export const MENU_TEMPLATE_PARTS_GENERAL = 'template-parts-general';
+
+export const TEMPLATE_PART_AREA_HEADER = 'header';
+export const TEMPLATE_PART_AREA_FOOTER = 'footer';
+export const TEMPLATE_PART_AREA_SIDEBAR = 'sidebar';
+
+export const TEMPLATE_PARTS_SUB_MENUS = [
+	{
+		area: TEMPLATE_PART_AREA_HEADER,
+		menu: MENU_TEMPLATE_PARTS_HEADERS,
+		title: __( 'Headers' ),
+	},
+	{
+		area: TEMPLATE_PART_AREA_FOOTER,
+		menu: MENU_TEMPLATE_PARTS_FOOTERS,
+		title: __( 'Footers' ),
+	},
+	{
+		area: TEMPLATE_PART_AREA_SIDEBAR,
+		menu: MENU_TEMPLATE_PARTS_SIDEBARS,
+		title: __( 'Sidebars' ),
+	},
+	{
+		area: null,
```

Comment:
Aligning this with changes in @Addison-Stavlo PR https://github.com/WordPress/gutenberg/pull/28410, would it make sense to assign `uncategorized` here too?

Reply:
`null` includes all template parts from the areas that aren't specified in this constant array. In this case template parts with `header`, `footer` or `sidebar` will be excluded from **General** but everything else will be included. This only matters if a theme has an area besides the header,footer and sidebar. So basically `null` is a catch-all, excluding items from the three other areas.

Reply:
What I was trying to say is that is that we are using `uncategorized` for the same thing (catch-all) on back end.

Reply:
Just checked the code you are right. I thought we allow other areas as well. I'll update the code to use `uncategorized`